nb-adapters/adapter-weaviate/src/main/resources/activities/weaviate_vector_live.yaml

```diff
@@ -0,0 +1,327 @@
+min_version: 5.21
+description: |
+  This is a template for live vector search testing.
+  Template Variables:
+
+  schema: Install the schema required to run the test
+  rampup: Measure how long it takes to load a set of embeddings
+  search: Measure how the system responds to queries while it
+   is indexing recently ingested data.
+  search: Run vector search with a set of default (or overridden) parameters
+  In all of these phases, it is important to instance the metrics with distinct names.
+  Also, aggregates of recall should include total aggregate as well as a moving average.
+
+scenarios:
+  weaviate_vectors:
+    delete_collection: >-
+      run tags==block:delete_collection
+      errors===stop
+      cycles===UNDEF threads===UNDEF
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+    schema_collection: >-
+      run tags==block:schema_collection
+      errors===stop
+      cycles===UNDEF threads===UNDEF
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+    get_collection_schema: >-
+      run tags==block:get_collection_schema
+      errors===stop
+      cycles===UNDEF threads===UNDEF
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+    # TODO - ,retry should be added to errors for rampup
+    rampup: >-
+      run tags==block:rampup
+      errors===warn,counter
+      cycles===TEMPLATE(train_cycles,TEMPLATE(trainsize,1000)) threads===TEMPLATE(train_threads,AUTO)
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+    count_vectors: >-
+      run tags==block:count_vectors
+      errors===stop
+      cycles===UNDEF threads===UNDEF
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+    search_points: >-
+      run tags==block:search_points
+      errors===warn,counter
+      cycles===TEMPLATE(testann_cycles,TEMPLATE(testsize,1000)) threads===TEMPLATE(testann_threads,AUTO)
+      uri=TEMPLATE(weaviatehost) token_file=TEMPLATE(token_file)
+
+params:
+  driver: weaviate
+  instrument: true
+
+bindings:
+  id_val: Identity();
+  id_val_uuid: ToHashedUUID() -> java.util.UUID
+  row_key: ToString()
+  row_key_batch: Mul(TEMPLATE(batch_size)L); ListSizedStepped(TEMPLATE(batch_size),long->ToString());
+  # filetype=hdf5 for TEMPLATE(filetype,hdf5)
+  test_floatlist_hdf5: HdfFileToFloatList("testdata/TEMPLATE(dataset).hdf5", "/test");
+  relevant_indices_hdf5: HdfFileToIntArray("testdata/TEMPLATE(dataset).hdf5", "/neighbors")
+  distance_floatlist_hdf5: HdfFileToFloatList("testdata/TEMPLATE(dataset).hdf5", "/distance")
+  # TODO - remove the local path
+  train_floatlist_hdf5: HdfFileToFloatList("local/testdata/TEMPLATE(dataset).hdf5", "/train");
```

Comment:
I've added `local` prefix to the directory path for my convenience. We can remove that later.